pkg/sqlcmd/sqlcmd.go

```diff
@@ -151,7 +151,7 @@ func (s *Sqlcmd) Run(once bool, processAll bool) error {
 				lastError = err
 			}
 		}
-		if err != nil && err != io.EOF && s.Connect.ExitOnError {
+		if err != nil && err != io.EOF && (s.Connect.ExitOnError && !s.Connect.IgnoreError) {
```

Comment:
Is this in line with the ODBC sqlcmd if -b is used and ':on error ignore' is used? What should take the precedence here?

Reply:
[https://microsoft.sharepoint.com/teams/sqltools/_layouts/OneNote.aspx?id=%2Fteams%2Fsqltools%2FSiteAssets%2FSQL%20Tools%20Notebook&wd=target%28SQL%20DB%20Emulator.one%7CACFE716C-A04C-4711-B338-012A51291AB8%2FGo%20Sql%20Cmd%20OnError%20Investigation%7C8D2F054E-1FC8-46A4-B2B1-08C3A98481A5%2F%29
onenote:https://microsoft.sharepoint.com/teams/sqltools/SiteAssets/SQL%20Tools%20Notebook/SQL%20DB%20Emulator.one#Go%20Sql%20Cmd%20OnError%20Investigation&section-id={ACFE716C-A04C-4711-B338-012A51291AB8}&page-id={8D2F054E-1FC8-46A4-B2B1-08C3A98481A5}&end](url)

Reply:
:on error ignore take precedence here . For more details, please refer document in description